src/config/scrivito.js

```diff
@@ -10,4 +10,8 @@ if (process.env.SCRIVITO_ENDPOINT) {
   config.endpoint = process.env.SCRIVITO_ENDPOINT;
 }
 
+if (process.env.SCRIVITO_PRERENDER === "true") {
```

Comment:
In other places `process.env.SCRIVITO_PRERENDER` is plainly used. See for example https://github.com/Scrivito/scrivito_example_app_js/blob/2e59bcef9a80a1278596eea48dd748174e496dda/webpack.config.js#L30

Given that I would prefer a stripped down version:

```suggestion
if (process.env.SCRIVITO_PRERENDER) {
```